shared/actions/signup/index.js

```diff
@@ -205,13 +216,13 @@ function signup (skipMail): TypedAsyncAction<Signup | ShowPaperKey> {
           skipMail
         },
         incomingCallMap: {
-          'keybase.1.loginUi.displayPrimaryPaperKey': ({sessionID, phrase}, {error, result}) => {
-            // TODO if the user doesn't Ack the paper key we should error
```

Comment:
the UI shouldn't let them continue, they can click back and cancel signup if they want